samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -100,7 +102,7 @@ LEDs:
 Building and running
 ********************
 
-.. |sample path| replace:: :file:`samples/bluetooth/mesh/light_lc`
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/light_ctrl`
```

Comment:
Btw, the `sample.yaml` also needs to be updated with a similar change.

Reply:
Thanks!